### ISSUE TITLE
Screen reader accessibility issue

### ISSUE THREAD

**Issue by sofquipeut** [n1d3v/naticord#31]
Hi,
I'm visually impaired using a screen reader (NVDA here).
I've installed Naticord and managed to log in using my token but cannot go further. I can see 2 tabs (Friends and servers) that I can access without difficulty using the keyboard,  but when I press Enter on one of the servers I'm subscribed to in the servers list, nothing happens. Does it mean that this is not accessible using the keyboard? Could you please try to fix this?

Thanks,

**Comment by n1d3v** [comment]
> Hi, I'm visually impaired using a screen reader (NVDA here). I've installed Naticord and managed to log in using my token but cannot go further. I can see 2 tabs (Friends and servers) that I can access without difficulty using the keyboard, but when I press Enter on one of the servers I'm subscribed to in the servers list, nothing happens. Does it mean that this is not accessible using the keyboard? Could you please try to fix this?
> 
> Thanks,

didnt think of this. im gonna try to fix it this in the future but yea i defo will
